angrybirds/mongo.py

```diff
@@ -0,0 +1,18 @@
+from pymongo import MongoClient
+
+
+def get_db():
+    client = MongoClient('localhost', 27017)
+    db = client['angry-birds']
+    return db
```

Comment:
All the `return`s in this file could be simplified by removing the variable assignment. I didn't do this as I thought assigning them to variables made the code more readable. I welcome input